src/pages/Home.vue

```diff
@@ -90,7 +93,8 @@ export default {
   },
   watch: {
     async activeWorkspace() {
-      if (this.isUserApproved) await this.fetchPlioIds();
+      if (this.isUserApproved)
+        await this.fetchPlioIds({ changedWorkspace: true });
```

Comment:
you can simply set `this.currentPageNumber = undefined;` here first and then call the function right?

Reply:
I think the best way to do this would be emit an event from the `WorkspaceSwitcher` component
Because we need to identify the case when the user has manually changed the workspace. There's a method in the `WorkspaceSwitcher` component called as `updateActiveWorkspace` which is invoked whenever the user switches the workspace manually.

We can emit an event there something like `"user-switched-workspace"` and listen to this event on `App.vue` and then unset the `currentPageNumber`

We should not do this inside the `watch` because there are other cases as well where the watch is called even when the user has not manually switched the workspace.

And I agree with aman that instead of passing a variable into `fetchPlioIds`, we should unset it right there and then

Reply:
Also please mention the issue in the PR description like `Fixes #399`

Reply:
I could emit an event from Workspace Switcher and capture the event in App, but since the page number manipulation happens in Home, using the store was a better choice. I created a variable in the generic module, userSwitchedWorkspace which will be set to false initially. When the user manually changes the workspace, the mutation is done, setting it to true. After all the plios for that workspace is fetched, the variable is set back to false.